.github/ISSUE_TEMPLATE/pre-submission.md

```diff
@@ -0,0 +1,145 @@
+name: Pre-submission Inquiry
+description: Make a pre-submission inquiry for an ontology. The purpose of a pre-submission inquiry is to announce an early ontology project that is not yet ready for review. It also serves as a reservation of an ontology namespace for 6 months.
+title: Pre-submission Inquiry for Ontology [NAME]
+labels: [ pre-submission ]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Do _NOT_ use this form to register a new ontology with the OBO Foundry. Please read the instructions provided here: http://obofoundry.org/docs/NewOntologyRegistrationInstructions.html.
+  - type: input
+    id: title
+    attributes:
+      label: Title
+      description: What is the full title of your ontology? Please do not include redundant acronyms.
+      placeholder: ex. Chemical Entities of Biological Interest
+    validations:
+      required: true
+  - type: input
+    id: slug
+    attributes:
+      label: Short Description
+      description: Please provide a short description of your ontology in one sentence.
+    validations:
+      required: true
+  - type: input
+    id: idspace
+    attributes:
+      label: Identifier Space
+      description: What is the prefix for the identifier space associated with your ontology? This is typically all capital letters. This *must* not conflict with any entries in the [Bioregistry](https://bioregistry.io/registry) nor [BioPortal](https://bioportal.bioontology.org). See OBO Principle [FP-003 "URI/Identifier Space"](https://obofoundry.org/principles/fp-003-uris.html).
```

Comment:
mention that it should be 3 or more letters? GO is perhaps not a good example to use if we want to discourage 2-letter identifier prefixes.